src/scenes/AcceleratorRouter/Applications/ApplicationsList.tsx

```diff
@@ -44,11 +72,48 @@ const defaultSearchOrder: SearchSortOrder = {
 
 const ApplicationList = () => {
   const dispatch = useAppDispatch();
+  const { activeLocale } = useLocalization();
   const snackbar = useSnackbar();
 
   const [requestId, setRequestId] = useState<string>('');
   const result = useAppSelector(selectApplicationList(requestId));
   const [applications, setApplications] = useState<ApplicationRow[]>([]);
+  const [countries, setCountries] = useState<Country[]>();
+
+  const featuredFilters: FilterConfig[] = useMemo(() => {
+    if (!activeLocale || !countries) {
+      return [];
+    }
+
+    const filters: FilterConfig[] = [
+      {
+        field: 'countryCode',
+        options: (countries || []).map((country) => country.code),
+        label: strings.COUNTRY,
+        renderOption: (id: string | number) => `${getCountryByCode(countries, id as string)}`,
+      },
+      {
+        field: 'type',
+        options: [strings.PRESCREEN, strings.APPLICATION],
+        label: strings.TYPE,
+      },
+    ];
+
+    return filters;
+  }, [activeLocale, countries]);
+
+  useEffect(() => {
+    if (activeLocale) {
+      const populateCountries = async () => {
+        const response = await LocationService.getCountries();
+        if (response) {
+          setCountries(response);
+        }
+      };
+
+      populateCountries();
+    }
+  }, [activeLocale]);
```

Comment:
This poor snippet of code is now in 7 locations

Reply:
Poor snippet indeed 😢